logging/cluster-logging-release-notes.adoc

```diff
@@ -100,4 +100,24 @@ As a workaround, if you change the secret, you can force the Fluentd pods to red
 $ oc delete pod -l component=fluentd
 ----
 
+[id="openshift-logging-5-2-0-deprecated-removed-features"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in OpenShift Logging and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments.
+
+[id="openshift-logging-5-2-0-legacy-forwarding"]
+=== Forwarding logs using the legacy Fluentd and legacy Syslog methods have been deprecated
```

Comment:
In some random OS docs syslog isn't capitalized (when it's both monospaced or plain), maybe I stumbled upon pages with older style requirements? E.g.
https://docs.openshift.com/container-platform/4.6/logging/cluster-logging-external.html

Not sure about OS docs guidelines, but in general the IBM guide recommends to highlight all method names with monospace.

Reply:
Thanks. Updated Syslog > syslog. In this case, we're using "method" in a general sense, not to signify something similar to a function in a programming language.